src/decode/lzma.rs

```diff
@@ -95,6 +113,9 @@ where
     LZB: lzbuffer::LZBuffer<W>,
 {
     _phantom: PhantomData<W>,
+    // buffer input data here if we need more for decompression, 20 is the max
+    // number of bytes that can be consumed during one iteration
```

Comment:
This comment mentions "20" but there is no 20 here. Here it should refer to `MAX_REQUIRED_INPUT`, and the explanation for 20 should be given in `MAX_REQUIRED_INPUT`'s definition.

Reply:
The value 20 was derived from the comment mentioned here:
https://github.com/OISF/libhtp/blob/0.5.x/htp/lzma/LzmaDec.h#L48

Reply:
What I meant is that there is no `20` in the code here, just `MAX_REQUIRED_INPUT`, so the comment should use `MAX_REQUIRED_INPUT`rather than `20`. The explanation for `20` should be given where the `MAX_REQUIRED_INPUT` constant is defined.